packages/block-library/src/columns/editor.scss

```diff
@@ -56,6 +42,11 @@
 			> .editor-block-list__block-edit {
 				margin-top: 0;
 				margin-bottom: 0;
+
```

Comment:
It looks like this change causes the drag handle and block mover buttons to now appear off-screen.

Before:
![Screen Shot 2019-03-26 at 5 06 57 pm](https://user-images.githubusercontent.com/677833/54984865-b8a0d380-4fea-11e9-9237-795e6f4a02d8.png)



After:
![Screen Shot 2019-03-26 at 5 12 59 pm](https://user-images.githubusercontent.com/677833/54984835-a4f56d00-4fea-11e9-8576-be66d5394759.png)

Reply:
> to now appear off-screen.

Could you provide the scenario under which this occurs? There are so many variations! Much appreciated.

@jasmussen I believe we discussed that it might be difficult to avoid this issue...?

Reply:
>  I believe we discussed that it might be difficult to avoid this issue...?

I can't recall the full context of that, been a busy few weeks, may have forgotten it. 

I think it's a fair point, and it seems like something worth looking into, especially if it's something that regresses in this branch compared to master. It's a bit hard to understand what might cause this just looking at the code — it's about the section block, but yet it touches some of the column block code here: https://github.com/WordPress/gutenberg/pull/13964/files#diff-49ac41bc72e24e343a451fe254801c0fL15

☝️ that up there is edgecase CSS intended to make those controls visible for a fullwide columns block, and I'm not sure why it's being removed as part of the container work being done. Not sure if it's what's causing the regression, but if not, we might actually want to duplicate this for the container block too.

Reply:
I'll look into this to ensure there is no regression of the Columns Block.

Reply:
I've pushed a fix which restores the fix only for width full Columns within a full width Section.

I believe however this is just plastering over the real issue which is that Gutenberg has a edge-to-edge canvas. Therefore when Blocks are full width it does not leave any space for move/drag handles. @talldan was going to raise a PR regarding this issue but it's not something we can address here so this fix is the best compromise.

Reply:
Thanks, and I agree, the comment in the code suggests one way to fix this.